### DISCUSSION TITLE
Nativescript `ns clean` Error : Unable to start Cleanup process. Error while reporting exception: Error: Unable to start Cleanup process

### DISCUSSION THREAD

**Discussion by pharshitha13** [NativeScript/NativeScript#10574] [Q&A]
Getting issue with command ns clean. ns build android or ns run android commands are not working.

Not able to build the project fails with this issue.

![ns_clean](https://github.com/NativeScript/NativeScript/assets/82640676/0feebc3c-c825-4733-93fd-c22fbcb48b0a)

It through errors Unable to start Cleanup process. Error while reporting exception: Error: Unable to start Cleanup process

Tried clearing temp, npm cache, restarted laptop, cloned as new project, checked with new bash. Nothing worked. Please help

**Comment by rigor789** [comment]
Try running with `--log trace` and share the logs, seems like it's unable to spawn a child process on your system.

**Reply by pharshitha13** [reply]
![nscleanlogs](https://github.com/NativeScript/NativeScript/assets/82640676/8f12a6b6-6628-4ca6-b455-17e4a83b21a1)

I ran ns clean --log trace
